### ISSUE TITLE
Missing getter for custom properties on StreamMetadata

### ISSUE THREAD

**Issue by wiegell** [EventStore/EventStore-Client-Go#178]
The only way to get customProperties seems to be as marshalled Json. Shouldn't there be a `GetCustomProperty` or `GetCustomProperties` method on `StreamMetadata` too?

```
type StreamMetadata struct {
	maxCount         []uint64
	maxAge           []time.Duration
	truncateBefore   []uint64
	cacheControl     []time.Duration
	acl              []interface{}
	customProperties map[string]interface{}
}
```

**Comment by YoEight** [comment]
@wiegell Thanks for reporting this.

The patch is available on version `v4.1.0`

**Comment by wiegell** [comment]
Thanks for the quick resolution!
